plugins/inspect/repositories_inspector.rb

```diff
@@ -100,8 +110,43 @@ def inspect(system, description, options = {})
       result = []
       summary = "Found 0 repositories."
     end
+    return RepositoriesScope.new(result), summary
+  end
 
-    description.repositories = RepositoriesScope.new(result)
-    summary
+  def inspect_yum_repositories(system)
+    yum_conf = IniFile.new(content: system.read_file("/etc/yum.conf"))
+    default_gpgcheck = yum_conf["main"]["gpgcheck"]
+    repositories = []
+    occurences = 0
+
+    # repositories can also be specified in yum.conf
+    repo_ini_list = [yum_conf]
+    system.run_command(
+      "find", "/etc/yum.repos.d/", "-type", "f", :stdout => :capture
+    ).split("\n").reject{ |item| item == "" }.each do |file|
+      begin
+        repo_ini_list << IniFile.new(content: system.read_file(file))
+      rescue(IniFile::Error)
+        Machinery::Ui.warn " -> The repository file '#{file}' couldn't be parsed."
+      end
+    end
+    repo_ini_list.each do |ini_file|
+      # filter the main section from yum.conf
+      ini_file.delete_section("main")
+      ini_file.each_section do |repo|
+        repositories << Repository.new(
+          alias:       repo,
+          name:        ini_file[repo]["name"],
+          type:        "rpm-md",
```

Comment:
I actually don't think it's out of scope. zypper and yum are already different enough to make it non-trivial to find a common model, so the cleaner approach would be to just store them as separate structures, similar to how we do it for the services as well. Then we would have a clean structure, which also would be capable of mixed repositories in case we ever want to have that.